sentry-okhttp/src/main/java/io/sentry/okhttp/SentryOkHttpEvent.kt

```diff
@@ -158,10 +157,12 @@ internal class SentryOkHttpEvent(private val hub: IHub, private val request: Req
 
         // We forcefully finish all spans, even if they should already have been finished through finishSpan()
         eventSpans.values.filter { !it.isFinished }.forEach {
-            // If a status was set on the span, we use that, otherwise we set its status as error.
-            it.status = it.status ?: SpanStatus.INTERNAL_ERROR
             moveThrowableToRootSpan(it)
-            it.finish()
+            if (finishDate != null) {
+                it.finish(it.status, finishDate)
```

Comment:
Shouldn't this be
```suggestion
                it.finish(it.status ?: SpanStatus.INTERNAL_ERROR, finishDate)
```

Reply:
it was, but it actually didn't make sense, at least in the case linked in this PR.
If the response is not read, there will be some running spans, and they don't have any error.
So they didn't have to be reported as errored in the first place, i guess

Reply:
If there is another possible use case, we can think more about it

Reply:
👍 makes a lot of sense this way then!